src/osemgrep/cli_scan/Scan_CLI.ml

```diff
@@ -479,6 +481,12 @@ let o_json : bool Term.t =
   in
   Arg.value (Arg.flag info)
 
+let o_incremental : bool Term.t =
+  let info =
+    Arg.info [ "incremental" ] ~doc:{|Output results incrementally.|}
```

Comment:
nit: `--incremental` is a bit vague, what do you think about something like `--incremental-output` instead?

Reply:
sure